src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -64,12 +65,10 @@
       final String acceptVersion, @Valid final MultipartFile file) {
     final String expectedAcceptVersion = storeApiVersion;
     if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
-      log.warn(
-          "Expected Accept-Version to be \"{}\" but was \"{}\". Only \"{}\" is currently supported.",
-          expectedAcceptVersion,
-          acceptVersion,
-          expectedAcceptVersion);
-      return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+      throw new UnsupportedOperationException(
+          String.format(
+              "Expected Accept-Version to be \"%s\" but was \"%s\". Only \"%s\" is currently supported.",
```

Comment:
Could use the constant in the string as well in place of `Accept-Version`. Looks odd to have expectedAcceptVersion twice in the message, but that is just a nit.

Reply:
I address that in https://github.com/connexta/ion-store/pull/212. This PR should not be merged until https://github.com/connexta/ion-store/pull/212 is merged.